packages/shared-ux/markdown/impl/markdown.tsx

```diff
@@ -35,19 +43,37 @@ export type MarkdownProps = Partial<
 
 export const Markdown = ({
   ariaLabelContent,
-  readOnly,
   markdownContent,
   children,
+  className,
   openLinksInNewTab = true,
   defaultValue = '',
   placeholder = '',
   height = 'full',
+  readOnly = false,
+  enableTooltipSupport = false,
+  validateLinks = false,
+  ...restProps
 }: MarkdownProps) => {
   const [value, setValue] = useState(defaultValue);
 
-  // openLinksInNewTab functionality from https://codesandbox.io/s/relaxed-yalow-hy69r4?file=/demo.js:482-645
-  const processingPlugins = getDefaultEuiMarkdownProcessingPlugins();
-  processingPlugins[1][1].components.a = (props) => <EuiLink {...props} target="_blank" />;
+  const { parsingPlugins, processingPlugins, uiPlugins } = getDefaultEuiMarkdownPlugins({
+    exclude: enableTooltipSupport ? undefined : ['tooltip'],
+  });
+
+  if (openLinksInNewTab) {
+    // overrides the default component defined for anchor tags
+    processingPlugins[1][1].components.a = (props) => <EuiLink {...props} target="_blank" />;
+  }
+
+  processingPlugins.push([rehypeRaw, {}], [rehypeSanitize, {}]);
```

Comment:
Add new plugins to support html tags in our markdown editor, to remain backwards compatible with previous markdown component.